### PR TITLE
Add quirk for Osram Smart+ Switch Mini

### PR DESCRIPTION
**Pull request by majkrzak**
According to https://gist.github.com/majkrzak/333417ea2d7e59f149c51e8016d10962

### CODE REVIEW

File: zhaquirks/osram/switchmini.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    BUTTON_2,
+    BUTTON_3,
+    LONG_RELEASE,
+    LONG_PRESS,
+    COMMAND_MOVE_TO_LEVEL_ON_OFF,
+    COMMAND_OFF,
+    COMMAND_MOVE,
+)
+
+OSRAM_DEVICE = 0x0810
```
Comment (dmulcahey) [inline zhaquirks/osram/switchmini.py:42]: Remove this and replace it with https://github.com/zigpy/zigpy/blob/29e362563f31bc8e77462ac5b2fbd78c40aa926f/zigpy/profiles/zha.py#L64

File: zhaquirks/osram/switchmini.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                    LightLink.cluster_id,
+                ],
+            },
+            # <SimpleDescriptor endpoint=2 profile=260 device_type=2064
+            # device_version=1
+            # input_clusters=[0, 4096, 64768]
+            # output_clusters=[3, 4, 5, 6, 8, 768, 4096]>
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: OSRAM_DEVICE,
```
Comment (dmulcahey) [inline zhaquirks/osram/switchmini.py:83]: Import https://github.com/zigpy/zigpy/blob/29e362563f31bc8e77462ac5b2fbd78c40aa926f/zigpy/profiles/zha.py#L64 and use that

File: zhaquirks/osram/switchmini.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                    LightLink.cluster_id,
+                ],
+            },
+            # <SimpleDescriptor endpoint=2 profile=260 device_type=2064
+            # device_version=1
+            # input_clusters=[0, 4096, 64768]
+            # output_clusters=[3, 4, 5, 6, 8, 768, 4096]>
+            3: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: OSRAM_DEVICE,
```
Comment (dmulcahey) [inline zhaquirks/osram/switchmini.py:101]: Same as above

File: zhaquirks/osram/switchmini.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    signature = {
+        MODELS_INFO: [(OSRAM, "Lightify Switch Mini")],
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=2064
+            # device_version=1
+            # input_clusters=[0, 1, 20, 4096, 64758]
+            # output_clusters=[3, 4, 5, 6, 8, 25, 768, 4096]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: OSRAM_DEVICE,
```
Comment (dmulcahey) [inline zhaquirks/osram/switchmini.py:58]: Import https://github.com/zigpy/zigpy/blob/29e362563f31bc8e77462ac5b2fbd78c40aa926f/zigpy/profiles/zha.py#L64 and use that

**Comment by majkrzak** [comment]
Great :roll_eyes: I've accidentally connected switch to Osram light bulbs and I'm not able to disconnect it. After reset, when trying to connect to hassio, it is successfully detected, but no single event is reported, but at least automation handlers are present..,

**Pull request merged**